### PR TITLE
Feat/export pdf

### PR DESCRIPTION
**Pull request by kopkop123**

### CODE REVIEW

File: backend/src/api/routes/page-route.ts [hunk 1]
```diff
@@ -236,4 +238,14 @@ router.post(
   run((req) => unpinPage(req.userId, req.params.id)),
 );
 
+router.post(
+  '/:id/download-pdf',
+  run((req) => downloadPDF(req.params.id)),
+);
```
Comment (mykytabatrak) [inline backend/src/api/routes/page-route.ts]: Shouldn't that be get?

**Review by mykytabatrak — DISMISSED** [review]

File: backend/src/common/utils/generate-pdf.util.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  const IS_PRODUCTION = process.env.NODE_ENV === 'production';
+  const md = new MarkdownIt();
+  const result = `<h1>${title}</h1>${md.render(content)}`;
+
+  let browser = null;
+  let file = null;
+
+  try {
+    browser = await (IS_PRODUCTION
+      ? puppeteer.connect({ browserWSEndpoint: 'wss://chrome.browserless.io/' })
```
Comment (AlikRakhmonov) [inline backend/src/common/utils/generate-pdf.util.ts]: Must be only connect option

**Review by AlikRakhmonov — CHANGES_REQUESTED** [review]

File: backend/src/api/routes/page-route.ts (hunk 1 above)
Comment (kopkop123) [reply backend/src/api/routes/page-route.ts]: fixed

File: backend/src/common/utils/generate-pdf.util.ts (hunk 2 above)
Comment (kopkop123) [reply backend/src/common/utils/generate-pdf.util.ts]: fixed

**Comment by AlikRakhmonov** [comment]
I do not see `.env.example` changes. You should have at least `PUPPETEER_SKIP_CHROMIUM_DOWNLOAD`

https://github.com/puppeteer/puppeteer/blob/v10.2.0/docs/api.md#environment-variables

**Review by AlikRakhmonov — DISMISSED** [review]
see my comment

**Review by AlikRakhmonov — APPROVED** [review]

**Review by mykytabatrak — APPROVED** [review]

**Pull request merged**